### PR TITLE
Lots of Ad Guide content updates

### PR DESCRIPTION
**Pull request by ericholscher**
This is mostly just a revamp,
along with some SEO focused keyword and content updates.

### CODE REVIEW

File: content/pages/learning-hub/advertiser-guide.md [hunk 1]
```diff
@@ -31,128 +38,134 @@ but it all starts with the image and text copy.
...
 
-Making sure your geographic targeting matches the location of your desired customers.
-Refining your scope by targeting a topic ensures your ad will be seen by the right audience.
 Developers come in all shapes, sizes, and [specialties](https://www.ethicalads.io/our-audience/),
 only some are a perfect fit for your product or service.
-Crafting highly targeted ads for each region of interest will help increase the CTR of your ads
+To get the most out of your campaign:
+
+* Make sure your geographic targeting matches the location of your desired customers.
+* Target a specific topic that relevant to your business.
```
Comment (davidfischer) [inline content/pages/learning-hub/advertiser-guide.md:67]:
```suggestion
* Target a specific topic that's relevant to your business.
```

File: content/pages/learning-hub/advertiser-guide.md [hunk 2]
```diff
@@ -31,128 +38,134 @@ but it all starts with the image and text copy.
...
 
-Our network supports ads with or without an image.
-Publishers choose the format that works best for their site,
-so you'll want to provide both formats so that your ad is shown
-on all relevant sites across the network.
-Sometimes if the image isn't conveying critical information to the ad,
-the same copy can be used both with and without an image.
-However, if your image contains important context or text,
-it's much better to craft a second text-only ad that can communicate your message on its own.
+When you run a campaign we recommend having 3-4 ad variations.
```
Comment (davidfischer) [inline content/pages/learning-hub/advertiser-guide.md:75]:
```suggestion
When you run a campaign we recommend having 3-4 ad variations especially for larger campaigns.
```

File: content/pages/learning-hub/advertiser-guide.md [hunk 3]
```diff
@@ -31,128 +38,134 @@ but it all starts with the image and text copy.
...
-## Examples of successful ads
+## Collaborate with us to get great results
 
-Here are some things that have worked well for our advertisers in the past:
+Our account managers will work with you to ensure the success of your campaign
+with our in-depth guidance:
+
+* **Initial audit**: Before your campaign launches we look over your ads and give ideas for ad iterations based on what we've seen work in the past.
+* **Two-week check in**: After we get click-through data on your ad variations, we conduct additional analysis to recommend which ads to disable, and new iterations of your highest performing ads.
+* **Campaign Wrapup**: At the end of your campaign, we will share an analysis of how your campaign performed overall, and suggestions for next steps to continue iterating on your marketing goals.
```
Comment (davidfischer) [inline content/pages/learning-hub/advertiser-guide.md:128]: This section is great. Should we mention video progress reports for larger campaigns?

**Review by davidfischer — APPROVED** [review]

File: content/pages/learning-hub/advertiser-guide.md (hunk 3 above)
Comment (ericholscher) [reply content/pages/learning-hub/advertiser-guide.md:129]: Yea, I feel like we should be selling this a lot more, but this was a good start.

I thought about it, but didn't want to over-promise for the default user. If we have an objective measure here (eg. accounts over $5,000/mo? then we should probably include it.)

Comment (davidfischer) [reply content/pages/learning-hub/advertiser-guide.md:129]: That's the cutoff where I'm doing them so I think that's worth including.

**Pull request merged**